openmdao/solvers/linesearch/tests/test_backtracking.py

```diff
@@ -656,7 +656,7 @@ def test_with_subsolves(self):
         ls = model.nonlinear_solver.linesearch = om.ArmijoGoldsteinLS(bound_enforcement='vector')
 
         # This is pretty bogus, but it ensures that we get a few LS iterations.
-        ls.options['c'] = 100.0
+        ls.options['c'] = 100.0  # FIXME c should be 0 <= c <= 1
 
```

Comment:
I will put up a story to fix this test so that `c` can be given an upper value of 1.0.